### DISCUSSION TITLE
Self made Linux game Server for the basement

### DISCUSSION THREAD

**Discussion by Mannshoch** [LizardByte/community#390] [Sunshine]
I need Hardware proposals for a Linux game-server. It is planed to run Wine on it with Windows Games between Windows 95 and Windows 2000, (they already run that way on my Desktop) also 0-ad.
I'm planing using it wit moonlight on a Tablet, Laptop and a Kodi (Raspberrypi 4)

What hardware do you propose for this usage.

**Comment by Loyola-London** [comment]
I have one of those. I bought a Precision 7810 (ebay ~300 at the time) and put 2x Xeon 2696 v3 (ebay ~200 at the time). The PC came with a single slot quadro card so I left that and added an RTX 3060 ti. I need the server for work things so I created a windows 10 VM with 16 threads leaving the remaining 56 threads for server stuff. I've had this setup since 2020 and it's been solid streaming everything from 720p on a tablet to 4k on a tv. The 7810's do not have native support for nvme, but you can buy add-in cards (ebay ~30 at the time) that will allow you to add 2-4 gen 3 nvme drives.

Ebay has great prices on retired dell servers/workstations and they are rock-solid linux performers.

**Reply by jimmt41** [reply]
I did something similar 6 yrs ago and over the past years I have slowly replaced everyhing to be more modern :) With that said Loyola advisement is an awesome recomendation. Not only is it a fun project, it's economical too so you are not out of much money. Also, as mentioned, it gives you a lab and game streaming all in one. I use mine for VM's, Plex, and gaming.

**Reply by noodlemctwoodle** [reply]
I migrated my entire desktop to a VM on my server, although the specs are probably a bit overkill for your needs. 

i9 13900k
96GB GSkill DDR5
ASUS 4080 Noctua Edition
ASUS WS680-ACE IPMI
3x Western Digital SN850X 4TB
2x Samsung 870 QVO 8TB
2x Seagate Iron Wolf Pro 22TB
2x Western Digital RED 14TB
2x Western Digital RED 12TB
14x Noctua NF-A12x25

![PXL_20231209_211811011.jpg](https://github.com/LizardByte/Sunshine/assets/15075388/e99a8cbc-ac40-4559-bdef-0a35afe8af02)

**Reply by Mannshoch** [reply]
And you work with Sunshine to access the VM?

**Reply by noodlemctwoodle** [reply]
@Mannshoch Yeah I have unRAID as my server OS and then my Gaming VM is Windows 11. I access all my games on the he VM using Sunshine and Moonlight on the clients.

**Reply by Mannshoch** [reply]
Nice.
I'm looking for knowledge about using Wine in this setting.
As an example I had this box in my Mind:
https://webshop.asus.com/de/Mini-PCs/Chromebox/ASUS-PN53-S5020MD-Mini-Desktop-PC/90MS02H1-M000M0

**Reply by Mannshoch** [reply]
Does linux need additional hardware if no display is connected? 
I sometimes read about strange additional hardware because of problems with virtual monitors. Is that also valid for AMD Ryzen on Linux?

**Reply by Loyola-London** [reply]
My linux server is 'headless' (i.e., no attached display) and it runs fine. The windows vm needs a dongle to simulate a display. I use this: https://a.co/d/eHw0OmA

I don't stream directly from linux, but my understanding is that sunshine streams what is sent to a display so you'd need some sort of real, fake (dongle), or virtual display to stream.

CPU shouldn't matter.

**Reply by noodlemctwoodle** [reply]
Yeah I am using a dummy HDMI adapter. They are cheap on Amazon

**Reply by Mannshoch** [reply]
why ist something like that needed?
And is Moonlight able to resize the screen to the screen e.g of my smartphone or is that not possible if an adapter is neede?
Or may could sunshine modify the resolution these adapters do fake?

[a comment by an automated account is not shown]
